pkg/kubelet/container/runtime.go

```diff
@@ -233,7 +233,6 @@ const (
 	ContainerStateExited  ContainerState = "exited"
 	// This unknown encompasses all the states that we currently don't care.
 	ContainerStateUnknown ContainerState = "unknown"
-	// Not in use yet.
 	ContainerStateCreated ContainerState = "created"
```

Comment:
Nit, if this is now used I'd prefer if it were ordered at the top to have `unknown` last and indicate how the lifecycle progresses